README.md

```diff
@@ -79,3 +79,56 @@ Extract the jar file and refer to the docs below.
 | Reladomo Architecture | Reladomo internal architecture. | architecture/ReladomoInternalArchitecture.html |
 | Presentations | Reladomo presentation materials. | presentations |
 
+
+## Acquiring Reladomo

```

Comment:
I don't want to have to change the readme every time we release. Just add a link to the releases tab in github and maybe a link to maven.org (searching on reladomo without version).